subsys/bootloader/Kconfig

```
@@ -116,6 +116,38 @@ config SB_DEBUG_SIGNATURE_PUBLIC_KEY_LAST
 	  first. This is meant to be used for testing looping through the
 	  public keys.
 
+config SB_MONOTONIC_COUNTER
+	bool "Enable HW monotonic version counter"
+	default y
+	help
+	  An array of slots is used for one counter. One slot is used per
+	  counter update, and each new slot must have a larger value than the
+	  previous. Application versions are checked against the current largest
+	  counter before being booted.
+
+config SB_NUM_MONOTONIC_COUNTERS
+	int "Number of monotonic counter slots"
+	default 240
+	range 1 300 if SOC_NRF5340_CPUAPP || SOC_NRF9160
+	range 1 1800 if SOC_SERIES_NRF52X
+	range 1 400 if SOC_SERIES_NRF51X
+	depends on SB_MONOTONIC_COUNTER
+	help
+	  The number of monotonic counter slots available for the counter.
+	  I.e. the number of times it can be updated.
+	  The slots are 16 bits each, and the slot array is preceded by a
+	  16 bit header. Because of this, the number of slots will be rounded
+	  up to the nearest odd number to ensure that header+slots is aligned on
+	  a 32 bit word.
+	  Rationale for the default number (240): Assume one update a month for
+	  10 years, then double that value just in case. This default fits
+	  comfortably within the "OTP" region of UICR.
+	  Regarding ranges: The actual maximum depends on the number of
+	  provisioned public keys since they shares space. The same is true if
+	  other data is stored in the "OTP" region (on nRF91 and nRF53).
```

```suggestion
	  other data is stored in the "OTP" region (on for example nRF91 and nRF53).
```

To future proof so that we don't have to update this list.